sections/sprint-5/limiting-transmissions.tex

```diff
@@ -0,0 +1,37 @@
+\section{Limiting the amount of UDP transmissions}
+As the game was being tested we encountered a problem with the amount UDP packages that were being transmitted.
+Initially there were no limitations to the number of packages sent, and the server would transmit as many as it possibly could.
+This would overburden the internet and could cause it to crash.
+A limit had to be set for how many packages were able to be transmitted per second.
+Since Pozyx can send updates on tags at a rate of approximately 60 per second, it would be preferable to have the limit at about the same as that.
+The limit could even be a bit higher than 60, since the updates are being transitted with UDP which means that there is no assurance that each package arrives.
```

Comment:
```suggestion
The limit could even be a bit higher than 60, since the updates are being transmitted with UDP which means that there is no assurance that each package arrives.
```